src/selectors/entities/posts.js

```diff
@@ -32,11 +32,22 @@ export function makeGetReactionsForPost() {
         getReactionsForPosts,
         (state, postId) => postId,
         (reactions, postId) => {
-            return Object.values(reactions[postId] || {});
+            return Object.values(reactions[postId] || []);
         }
     );
 }
 
+const getReactionsForPost = makeGetReactionsForPost();
+
+export function getUserIdsFromReactions(state, postId) {
+    const reactions = getReactionsForPost(state, postId);
+
+    return reactions.reduce((acc, reaction) => {
```

Comment:
Isn't that reduce() in the end doing the same as `reactions.map(reaction => reaction.user_id)`?

Reply:
Oh right, I'll update that.  Thanks!